source/codegen/metadata/nidmm/enums.py

```diff
@@ -149,51 +158,64 @@
             }
         ]
     },
-    'CompensatedOhms': {
-        'values': [
-            {
-                'name': 'NIDMM_VAL_OFFSET_COMP_OHMS_OFF',
-                'value': 0
-            },
-            {
-                'name': 'NIDMM_VAL_OFFSET_COMP_OHMS_ON',
-                'value': 1
-            }
-        ]
-    },
     'Configuration4022': {
+        'codegen_method': 'public',
         'values': [
             {
-                'name': 'NIDMM_4022_CONFIG_GUARD',
```

Comment:
Working on straightening these names out with [this PR](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/361682)